debian/control

```diff
@@ -0,0 +1,16 @@
+Source: youtube-dl-gui
+Section: web
+Priority: optional
+Maintainer: nodiscc <nodiscc@gmail.com>
+Build-Depends: debhelper (>= 9), python-all
+X-Python-Version: >= 2.7
+Standards-Version: 3.9.8
+Homepage: https://github.com/MrS0m30n3/youtube-dl-gui
+
+Package: youtube-dl-gui
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends}, ${python:Depends}
```

Comment:
Maybe we should also add the `Recommends: ffmpeg`

Reply:
Done https://github.com/MrS0m30n3/youtube-dl-gui/pull/189/files#diff-d837a1c17de0268bcea321239ddbed47R13